pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {name = "Tom Hu", email = "thomas.hu@sentry.io"},
+]
+maintainers = [
+    {name = "Codecov Support", email = "support@codecov.io"},
+]
+description = "Codecov Command Line Interface"
+dependencies = [
+    "click==8.*",
+    "httpx==0.27.*",
+    "ijson==3.*",
+    "pyyaml==6.*",
+    "regex",
+    "responses==0.21.*",
+    "sentry-sdk>=2.20.0",
+    "test-results-parser==0.5.*",
+    "tree-sitter==0.20.*",
+    "wrapt>=1.17.2",
+]
+license = {file = "LICENSE"}
+name = "codecov-cli"
+readme = "README.md"
+requires-python = ">= 3.9"
+version = "10.0.1"
+
+[project.scripts]
+codecov = "codecov_cli.main:run"
+codecovcli = "codecov_cli.main:run"
+
+[tool.setuptools]
+packages = ["codecov_cli"]
+
+[tool.cibuildwheel]
```

Comment:
Future improvement: add testing configuration per https://cibuildwheel.pypa.io/en/stable/options/#testing. This will run tests with each of the built wheels for each platform right after building them.
It may be the last full pre-release test run against the exact artifacts you're about to upload to PyPI.

Reply:
https://github.com/codecov/codecov-cli/issues/610